bin/dependency-check-pr.sh

```diff
@@ -124,7 +126,33 @@ ${PR_NOTE}"
       echo "No commits found for diff."
     fi
   fi
-  echo "✨ Done"
+  echo -e "✨ ${green}Done${reset} ✨"
+}
+
+# Get the latest tag from the source. If source is undefined, default to "github".
+# Usage example: LATEST_TAG=$(get_latest_tag "mongodb-php-library" "pecl")
+get_latest_tag() {
+  local repo="$1"
+  local source="$2"
+  local LATEST_TAG
+
+  # We're defaulting to GitHub, but we want to check against releases AND tags.
+  if [[ "${source}" == "github" || "${source}" == "null" ]]; then
```

Comment:
"null" is a weird yq-ism, I'd make this `|| ""` within the function as well for cleaner coverage.
```suggestion
  if [[ "${source}" == "github" || "${source}" == "null" || "${source}" == "" ]]; then
```